lib/control-connection.js

```diff
@@ -25,10 +25,11 @@ var schemaChangeTypes = {
 /**
  * Represents a connection used by the driver to receive events and to check the status of the cluster
  * @param {Object} options
+ * @param {ProfileManager} profileManager
  * @extends EventEmitter
  * @constructor
  */
-function ControlConnection(options) {
+function ControlConnection(options, profileManager) {
```

Comment:
My intention was to use `this.profileManager.getDefaultLoadBalancingPolicy()` but it looks I unintentionally kept the reference to `options.policies.loadBalancing` on line 43.

The question is: should we use the lb policy defined in the options (`options.policies.loadBalancing`) or the one in the default profile? Most of the time should be the same... 
But if not, what does it mean to have different lb policies (options.policies vs default profile)... I think the one in the default profile should be used.